test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1132,8 +1153,8 @@ describe('diffConversationCallStates()', () => {
   });
 
   it('should return "true" if one call state is missing', () => {
-    expect(handler.diffConversationCallStates(undefined, call2)).toBe(true);
-    expect(handler.diffConversationCallStates(call1, undefined)).toBe(true);
+    expect(handler.diffConversationCallStates(undefined as any, call2)).toBe(true);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(handler.diffConversationCallStates(undefined as unknown, call2)).toBe(true);
```